### DISCUSSION TITLE
GitHub code is behind server code

### DISCUSSION THREAD

**Discussion by militaruCD** [community/community#24027] [New to GitHub]
<p>Hi guys,</p>
<p>I’ve used  GitHub to deploy code on my website (hosted on DigitalOcean and managed with Ploi).</p>
<p>I’ve uploaded (via GitHub) a script bought from CodeCanyon to my server, but lately, I’ve installed some addons and updates to the said script.</p>
<p>I want to bring someone on board to make some changes and add some features, the idea was to create a separate branch for them to work on. But I now realise that the code on my GitHub is different from the one that’s live on the server ( added updates and installed extra modules).</p>
<p>My question is:<br>
How can I synchronise my GitHub repository(left behind) with the code that’s on the website (went ahead with changes)?</p>
<p>Thank you for your advice,<br>
C</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>The easiest option is probably to clone your Github repo somewhere and copy the files from the server over it. Then check the changes (including new files) to see what you want or don’t want to keep. Commit what you want to keep, and push to the Github repo.</p>

**Comment by militaruCD** [comment]
<p>That’s the way we did it. Thank you for your help. have a great weekend.</p>
